### ISSUE TITLE
[javascript] Prohibit any console methods

### ISSUE THREAD

**Issue by dschach** [pmd/pmd#5105]
**Proposed Rule Name:** 
AvoidConsoleStatements

**Proposed Category:** 
Performance or Security

**Description:**
Use of console.log or console.error can expose sensitive data and is a performance hit. We have a rule for this in Apex, and should have a similar one for JavaScript.

**Code Sample:** This should include code, that should be flagged by the rule. If possible, the "correct" code
according to this new rule should also be demonstrated.

```javascript
console.log('foo'); // bad
console.error('foo'); // bad
```

**Possible Properties:**

*   Should this rule be customizable via properties?
Perhaps we specify the kinds of console statements we block? error vs log vs ...

**Context:**
* Related Apex Rule: [AvoidDebugStatements](https://docs.pmd-code.org/latest/pmd_rules_apex_performance.html#avoiddebugstatements)
